dot/parachain/service.go

```diff
@@ -10,14 +10,14 @@ import (
 	"github.com/ChainSafe/gossamer/dot/network"
 	availability_store "github.com/ChainSafe/gossamer/dot/parachain/availability-store"
```

Comment:
noticing this now, you could change this too.

```suggestion
	availabilitystore "github.com/ChainSafe/gossamer/dot/parachain/availability-store"
```

Reply:
Updated.